server/app.js

```diff
@@ -260,17 +261,19 @@ const createApp = services => {
 
   // eslint-disable-next-line no-unused-vars
   function renderErrors(error, req, res, next) {
+    if (error instanceof NotFound) {
+      logger.warn(`Failed to find: ${error.message}`);
+      logger.debug(error.stack);
+      res.status(404);
+      return res.render('pages/404');
+    }
+
     logger.error(`Unhandled error - ${req.originalUrl} - ${error.message}`);
     logger.debug(error.stack);
-    res.status(error.status || 500);
```

Comment:
generally shouldn't propagate downstream http statuses to the browser - they should generally be interpreted as 500s unless actually handled